ircd/listener.c

```diff
@@ -656,7 +661,11 @@ accept_precallback(rb_fde_t *F, struct sockaddr *addr, rb_socklen_t addrlen, voi
 
```

Comment:
`check_reject` above will add the client to a list; code in ircd/reject.c will later send error messages without SSL negotiation as well. Fitting in with the theme of this pull request, those should be replaced with TLS alerts as well.